docs: better docs for docker & default logging (#459)
eduMFA/eduMFA: doc/installation/docker.rst

.. code-block:: cfg
   
   # The realm, where users are allowed to login as administrators
   SUPERUSER_REALM = ['super', 'administrators']
   # Your database
   SQLALCHEMY_DATABASE_URI = 'mysql+pymysql://edumfa:pass@mariadb/edumfa'
   # This is used to encrypt the auth_token
   SECRET_KEY = 'strong-key'
   # This is used to encrypt the admin passwords
   EDUMFA_PEPPER = "strong-pepper"
   # This is used to encrypt the token data and token passwords
   EDUMFA_ENCFILE = '/etc/edumfa/enckey'
   # This is used to sign the audit log
   EDUMFA_AUDIT_KEY_PRIVATE = '/etc/edumfa/private.pem'
   EDUMFA_AUDIT_KEY_PUBLIC = '/etc/edumfa/public.pem'

.. code-block:: yaml

   version: '3'
   services:
      mariadb:
         image: docker.io/mariadb:lts-jammy
         restart: always
         volumes:
            - maria-data:/var/lib/mysql:rw
         environment:
            - MARIADB_PORT_NUMBER=3306
            - MARIADB_DATABASE=edumfa
            - MARIADB_USER=edumfa
            - MARIADB_PASSWORD=pass
            - MARIADB_ROOT_PASSWORD=pass
      edumfa:
         image: ghcr.io/edumfa/edumfa:latest
         ports:
            - "8000:8000"
         volumes:

>             - edumfa-data:/etc/edumfa

Reviewer: ```suggestion
            - edumfa-config:/etc/edumfa
```

            - /path/to/edumfa.cfg:/etc/edumfa/edumfa.cfg
         environment:
            - EDUMFA_ADMIN_USER=admin
            - EDUMFA_ADMIN_PASS=Passwort123
         depends_on:
            - mariadb

   volumes:

>       edumfa-data:

Reviewer: ```suggestion
      edumfa-config:
```

- Map port 8000 on the host to port 8000 in the container (`-p 8000:8000`)
- Name the container "edumfa" (`--name edumfa`)

Persistent Data 
...............

To persist data between container restarts, you can mount a volume for the database:

.. code-block:: bash

   docker run -d -p 8000:8000  -v /path/to/edumfa.cfg:/etc/edumfa/edumfa.cfg -v edumfa_data:/etc/edumfa --name edumfa ghcr.io/edumfa/edumfa:latest

> This will create a named volume `edumfa_data` that will persist your eduMFA data. This volume will contain the encryption key and the audit key.

Reviewer: ```suggestion
This will create a named volume `edumfa_config` that will persist your eduMFA configuration. This volume will contain the encryption key and the audit key.
```

   docker pull ghcr.io/edumfa/edumfa:latest
   docker stop edumfa
   docker rm edumfa

>    docker run -d -p 8000:8000  -v /path/to/edumfa.cfg:/etc/edumfa/edumfa.cfg -v edumfa_data:/etc/edumfa --name edumfa ghcr.io/edumfa/edumfa:latest

Reviewer: ```suggestion
   docker run -d -p 8000:8000  -v /path/to/edumfa.cfg:/etc/edumfa/edumfa.cfg -v edumfa_config:/etc/edumfa --name edumfa ghcr.io/edumfa/edumfa:latest
```
